### PR TITLE
Make design changes to match figma

### PR DESCRIPTION
**Pull request by SashankBalusu**
[//]: # "Feel free to customize this template to your project's vibes"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- I mainly updated the styling to better match the figma design
- I arranged the components using a flex dispaly to better align them
- I also ensured that the image was showing and font sizes and colors were relatively symmetric to the design

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
- I think the best order to review is probably the app.tsx file and then the associated styling
- This can be reviewed all at once

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

https://stackoverflow.com/questions/8129043/css-making-a-child-div-elements-height-and-width-x-pixels-smaller-than-the-par
https://www.w3schools.com/tags/tag_hr.asp
https://stackoverflow.com/questions/73361987/how-to-use-an-image-from-a-path-in-react-typescript-without-import-or-require
https://stackoverflow.com/questions/64254356/how-to-make-padding-not-apply-to-one-element
https://tiloid.com/p/adding-multiple-classes-to-a-component

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @insert pl github username here

### CODE REVIEW

File: app/styles.module.css [hunk 1]
```diff
@@ -13,4 +13,73 @@
...
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  background: #fafafa;
+  margin: -10px;
+  margin-bottom: 10px;
+}
```
Comment (ronniebeggs) [inline app/styles.module.css:74]: Instead of using a negative margin value to undo the padding of `.content`, consider moving the `.header` so it doesn't inherit the padding styles. We generally want to avoid negative/inverted spacing values.

File: app/page.tsx [hunk 2]
```diff
@@ -8,41 +8,67 @@ export default function Home() {
...
+            <a style={{ color: 'blue' }} href="https://www.codetenderloin.org">
+              https://www.codetenderloin.org/
+            </a>
+          </p>
+          <img
+            style={{
+              width: '-webkit-fill-available',
+              borderRadius: '8px',
+              marginTop: '2%',
+            }}
```
Comment (ronniebeggs) [inline app/page.tsx:39]: Let's move these inline styles to a separate CSS class so our styling convention is consistent.

File: app/styles.module.css [hunk 3]
```diff
@@ -13,4 +13,73 @@
...
+  margin-top: 3%;
+  display: flex;
+  flex-direction: row;
+  padding: 5px;
+}
+.postInteractions > p {
+  font-size: 11px;
+  font-weight: 600;
+  padding: 1%;
+  margin-left: 1%;
```
Comment (ronniebeggs) [inline app/styles.module.css:56]: Let's avoid applying padding, margin, or general spacing styles directly to text. See if you can apply these styles from an outer div if possible.

File: app/page.tsx [hunk 4]
```diff
@@ -8,41 +8,67 @@ export default function Home() {
...
+            <p>256 Likes</p>
+            <ShareIcon size={24} />
+          </div>
+        </div>
 
-        <p>
-          Image Link:
-          https://cdn.britannica.com/51/178051-050-3B786A55/San-Francisco.jpg
-        </p>
+        <hr style={{ marginTop: '4%', marginBottom: '4%' }}></hr>
```
Comment (ronniebeggs) [inline app/page.tsx:49]: This divider doesn't span the full width of the screen. Consider restructuring divs so either:
- The divider separates "post" and "comments" divs.
- Or, we get rid of the divider, apply a border top/bottom style to the adjacent divs.

File: app/styles.module.css [hunk 5]
```diff
@@ -13,4 +13,73 @@
...
+  padding-top: 10px;
+  padding: 5px;
+}
+
+div.headingInfo p:last-of-type {
+  margin-left: auto;
+  color: lightgray;
+  font-size: 12px;
+  font-weight: normal;
+}
```
Comment (ronniebeggs) [inline app/styles.module.css:38]: Can we use `justify-content: space-between` to separate the profile/username from the post date. And `align-items: center` to align items horizontally, that way we don't need to apply padding styles.

File: app/styles.module.css [hunk 6]
```diff
@@ -13,4 +13,73 @@
...
+  color: #0078e8;
+}
+.header > h1 {
+  font-size: 14px;
+  font-weight: 600;
+}
+
+.comment {
+  padding-left: 30px;
 }
```
Comment (ronniebeggs) [inline app/styles.module.css:85]: Consider whether it's possible to have the username and comment text in the same div, that way they're always in line with each other.

**Review by ronniebeggs — CHANGES_REQUESTED** [review]
Looking good so far! Generally, I think it's cleanest to define styles for each distinct functional element, instead of having conditional CSS styles, that way you can more easily reuse styles across different parts of your site (I'm slightly opinionated tho). 

Try addressing a few of these comments before the workshop tomorrow, and we can discuss questions as a group!
